### DISCUSSION TITLE
Watch screen lighting always on, during sports

### DISCUSSION THREAD

**Discussion by YokoTheDuke** [zepp-health/zeppos-forum#335] [User Feedback]
It would be an excellent setting option if you could use the ZEPP app to set whether the display stays lit while running, cycling or whatever during sporting activities. I like to run, but I have to move my wrist up to 3 times, or rather "wag" it back and forth until the running values can finally be seen. That's annoying. My watch is the Amazfit Ultra!
What is your opinion on the idea? I hope there isn't a setting that I missed.

**Comment by MyoungXUE** [comment]
Hello @YokoTheDuke . You can refer to this path for settings.

Workour -> Workout setting -> Wake During Workout

**Reply by YokoTheDuke** [reply]
Hello @MyoungXUE,

oh, that's very nice. Thank you very much for this helpful answer. I did not knew that. Now I can also see that I can even turn the display on throughout the day, not just during workouts. So I no longer have any wishes at all. Thank you for your great app on iOS and on the Amazfit Ultra, the good GUI and now also the top UX. Thanks Sir and Keep on rockin'
